Lib/test/test___all__.py

```diff
@@ -30,27 +30,21 @@ def check_all(self, modname):
             raise NoAll(modname)
         names = {}
         with self.subTest(module=modname):
-            with support.check_warnings(
```

Comment:
@1st1 can the changes from lines 33-53 remain? This is unrelated to the streaming API.